display.h

```diff
@@ -1,5 +1,12 @@
 #include "util.h"
 
+void showGameIntroductionStaticStart();
```

Comment:
There is no reason to declare twice those methods since that module it's used just to declare methods, not use them.